pkg/management/component.go

```diff
@@ -589,15 +589,24 @@ func (c *component) ExecuteActionsEmail(ctx context.Context, realmName string, u
 	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
 
 	var actions = []string{}
-
+	var eventRecorded bool = false
```

Comment:
`var eventRecorded = false` should be enough !?

Reply:
Should be enough, no? Do you see a problem here ?